Makefile

```
@@ -75,12 +75,22 @@ install:
 	mkdir --parents $(SERVICEPATH)
 	cp --force $(service) $(SERVICEPATH)
 	@echo ""
-
+	
 	@echo "# Copying default configuration file to $(CONFIGPATH)/$(config)"
 	mkdir --parents $(CONFIGPATH)
 	-cp --no-clobber $(config) $(CONFIGPATH)
 	@echo ""
 
+	@echo "# Do you want to add the configuration with current all keyboards?(y/N)"
+	@read -r answer; \
+	if [ "$$answer" = "y" ] || [ "$$answer" = "Y" ]; then \
+		echo "\nUpdating configuration with all keyboards..."; \
+		bash ./scripts/update_conf_with_all_keyboards.sh $(CONFIGPATH)/$(config); \
+	else \
+		echo "\nSkipping update conf"; \
+	fi
+	@echo ""
```

Suggestion on the grammar here:
```suggestion
	@echo "# Do you want to add all currently connected keyboards to your configuration? (y/N)"
	@read -r answer; \
	if [ "$$answer" = "y" ] || [ "$$answer" = "Y" ]; then \
		echo "\nUpdating configuration..."; \
		bash ./scripts/update_conf_with_all_keyboards.sh $(CONFIGPATH)/$(config); \
	fi
	@echo ""
```

Any reason for the capital `N` in the message (at `(y/N)`)?

---

I totally agree with the grammar and sentence feedback. As a non-native English speaker, this kind of feedback is extremely valuable to me.

The reason 'N' is capitalized is to indicate to users that the default value is 'N'.

---

Thanks for the explanation. This makes sense to me.